markdown-mode.el

```diff
@@ -8451,39 +8510,6 @@ See `markdown-wiki-link-p' and `markdown-follow-wiki-link'."
       (markdown-follow-wiki-link (markdown-wiki-link-link) arg)
     (user-error "Point is not at a Wiki Link")))
 
-(defun markdown-highlight-wiki-link (from to face)
```

Comment:
Please don't remove public interfaces, this causes breaking backward compatibility